master/buildbot/status/web/build.py

```diff
@@ -483,10 +415,6 @@ def stop(self, req, auth_ok=False):
     def stopchain(self, req):
         return StopBuildChainActionResource(self.build_status)
 
-    def rebuild(self, req):
```

Comment:
`if path == "rebuild":  ` donw needs to go away as well https://github.com/Unity-Technologies/katana/pull/290/files#diff-4f395e7327ec18b3373c906998f8ae3aR428